src/Xamarin.Android.Build.Tasks/Tasks/R8.cs

```diff
@@ -139,6 +122,23 @@ protected override CommandLineBuilder GetCommandLineBuilder ()
 				tempFiles.Add (temp);
 				cmd.AppendSwitchIfNotNull ("--pg-conf ", temp);
 			}
+			if (!string.IsNullOrEmpty (ProguardConfigurationFiles)) {
```

Comment:
I think  you might need to adjust which files are passed in by default:
```
BuildHasNoWarnings(False,True,True,"apk")
...
Should have no more than 1 MSBuild warnings.
...
(_CompileToDalvik target) -> 
  obj/Debug/proguard/proguard_xamarin.cfg : warning XA4304: ProGuard configuration file 'obj/Debug/proguard/proguard_xamarin.cfg' was not found.
```
I would guess that we are putting `proguard_xamarin.cfg` in the item group always, and we need to only do that when `$(AndroidLinkTool)` isn't blank?